Makefile

```
@@ -0,0 +1,32 @@
+.PHONY: all build clean copy-src local requirements setup test up update-dependencies
+
+CKAN_HOME := /srv/app
+
+all: build
+
+build:
+	docker-compose build
+
+clean:
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml down -v --remove-orphans
+
+copy-src:
+	docker cp catalog-app_ckan_1:$(CKAN_HOME)/src .
+
+dev:
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml build
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
```

```suggestion
	docker-compose -f docker-compose.dev.yml build
	docker-compose -f docker-compose.dev.yml up
```
Both files uses port 5000 with different service's name (ckan and ckan-dev)
I get a conflict there. Maybe we re-check the use of this two files togheter.